config/resources/parliament-domain.json

```diff
@@ -0,0 +1,91 @@
+{
+  "version": "0.1",
+  "prefixes": {
+    "parl": "http://mu.semte.ch/vocabularies/ext/parlement/",
+    "dossier": "https://data.vlaanderen.be/ns/dossier#",
+    "adms": "http://www.w3.org/ns/adms#",
+    "prov": "http://www.w3.org/ns/prov#"
+  },
+  "resources": {
+    "parliament-flows": {
+      "name": "parliament-flow",
+      "class": "parl:Parlementaireaangelegenheid",
+      "attributes": {
+        "opening-date": {
+          "type": "date",
+          "predicate": "dossier:openingsdatum"
+        },
+        "closing-date": {
+          "type": "date",
+          "predicate": "dossier:sluitingsdatum"
+        },
+        "parliament-id": {
+          "type": "string",
+          "predicate": "parl:id"
+        }
+      },
+      "relationships": {
+        "status": {
+          "predicate": "adms:status",
+          "target": "concept",
+          "cardinality": "one"
+        },
+        "parliament-subcase": {
+          "predicate": "parl:parlementaireProcedurestap",
+          "target": "parliament-subcase",
+          "cardinality": "one"
+        }
```

Comment:
Will add a relationship to `case` here through `parl:behandeltDossier` as mentioned in my other comment